lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -58,6 +60,7 @@ import { JWT } from '../utils/jwt';
  * @typedef {object} ContentInfoConfig
  * @prop {ContentInfoLogo} logo - Logo of the content provider
  * @prop {ContentInfoItem} item
+ * @prop {ContentInfoItem} container
```

Comment:
These changes add a `container` object, but leave the `containerTitle` in place until changes can be made on the client side (otherwise we'd break the client).